spec/features/admin/new_user_registration_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'New user registration' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.